tests/foreman/api/test_contentviewversion.py

```diff
@@ -298,6 +298,54 @@ def test_positive_delete_non_default(self):
         self.assertEqual(len(content_view.read().version), 0)
 
     @tier2
+    def test_positive_delete_composite_version(self):
+        """Create composite content view and publish it. After that try to
+        disassociate content view from 'Library' environment through
+        'delete_from_environment' command and delete content view version from
+        that content view. Add repository to initial content view
+        for better coverage.
+
+        :id: b5bb547e-0174-464c-b974-0254d372cdd6
+
+        :expectedresults: Content version deleted successfully
+
+        :CaseLevel: Integration
+
+        @BZ: 1276479
+        """
+        org = entities.Organization().create()
+        # Create and publish product/repository
+        product = entities.Product(organization=org).create()
+        repo = entities.Repository(
+            product=product, url=FAKE_1_YUM_REPO).create()
+        repo.sync()
+        # Create and publish content views
+        cv_versions = []
+        for _ in range(2):
```

Comment:
I've repeated steps from verification comment but looks like indeed only one CV would be enough.